content/en/docs/quickstart.md

```diff
@@ -43,86 +43,90 @@ notation list --plain-http $IMAGE
+Use `notation certificate list` to confirm whether the certificate is stored in the trust store.
 
 ```console
-notation list --plain-http $IMAGE
+notation cert list
 ```
 
-Confirm there is one listed, for example:
+## Sign the container image
 
-```output
-$ notation list --plain-http $IMAGE
-sha256:1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef
-```
+Use `notation sign` to sign the container image with COSE signature format.
 
-## Verify the signature of an image
+```console
+notation sign --plain-http --signature-format cose $IMAGE
+```
 
-Use `notification verify` to verify any signatures on your container image.
+Use `notation list` to show the signatures associated with the container image.
 
 ```console
-notation verify --plain-http $IMAGE
+notation list --plain-http $IMAGE
 ```
 
-Confirm your container image fails validation. For example:
+Confirm there is one signature, for example:
 
 ```output
-$ notation verify --plain-http $IMAGE
-Error: trust certificate not specified
-2022/09/30 15:24:14 trust certificate not specified
+$ notation list --plain-http $IMAGE
+localhost:5000/net-monitor@sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a
+└── application/vnd.cncf.notary.v2.signature
+    └── sha256:ba3a68a28648ba18c51a479145fca60d96b43dc96c6ab22f412c89ac56a9038b
 ```
 
-This verification fails because you have not trusted the public key for your container image.
+## Create a trust policy
 
-## Add a trusted public key
+In order to verify the container image, you need to configure the trust policy to specify trusted identities which sign the artifacts, and level of signature verification to use. The trust policy is a JSON document with the name `trustpolicy.json` under `$HOME/.config/notation/`.
 
-Add the public from the certificate you generated earlier as a trusted certificate.
+Once the trust policy file is executed as below, it creates one trust policy named `wabbit-networks-images`. This policy only applies to the repository `$REPO`. The level of signatureVerification is `strict`, which enforces a full validation. Any validation failure will fail the whole signature verification process. The trust store used for this policy is named `wabbit-networks.io` of type `ca`, which is created in previous step. Use `notation cert show` to get trust identity info from the subject field of the self-signed certificate.
 
 ```console
-notation cert add --name "wabbit-networks.io" <EXAMPLE_PATH>/notation/localkeys/wabbit-networks.io.crt
+cat <<EOF > $HOME/.config/notation/trustpolicy.json
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "wabbit-networks-images",
+            "registryScopes": [ "$REPO" ],
+            "signatureVerification": {
+                "level" : "strict" 
+            },
+            "trustStores": [ "ca:wabbit-networks.io" ],
+            "trustedIdentities": [
+                "x509.subject: CN=wabbit-networks.io,O=Notary,L=Seattle,ST=WA,C=US"
```

Comment:
Please use: `"trustedIdentities": ["*"]` instead so it is an easier reference for people to build.